extensions/integration-tests/setEnvironmentVariables.js

```diff
@@ -0,0 +1,141 @@
+const path = require('path');
+const os = require('os');
+const fs = require('fs');
+
+/**
+ * Launch options
+ * This script will launch a new process with environment variables required to run the test cases
+ * below are launch options, feel free to contribute new options here
+ */
+const LAUNCH_TERMINAL_WINDOWS = 'start cmd.exe';
+const LAUNCH_TERMINAL_MAC = 'open -a Terminal -n';
+const LAUNCH_GIT_BASH_WINDOWS = '"C:\\Program Files\\Git\\git-bash.exe"';
```

Comment:
Should check that this exists and output a useful error message on startup if it doesn't.

Reply:
done